README.md

```diff
@@ -58,7 +58,7 @@ environment can also be replicated using Vagrant.
 1. ````$ vagrant plugin install vagrant-berkshelf````
 3. ````$ vagrant up````
 
-As you'll see in our ````Berksfile```` file, our Vagrant instance uses cookbooks from a number of places.  Cookbooks that are specific to OpenStax and OpenStax Accounts live [on github](https://github.com/openstax/openstax_cookbooks).  If you find yourself needing to modify these cookbooks, clone the cookbook repository and then set the ````OPENSTAX_COOKBOOKS_PATH```` environment variable to the absolute path of the checked out cookbooks before running vagrant commands.  This will cause Vagrant and Berkshelf to use your local copy of the OpenStax cookbooks instead of the ones on Github. 
+As you'll see in our ````Berksfile```` file, our Vagrant instance uses cookbooks from a number of places.  Cookbooks that are specific to OpenStax and OpenStax Accounts live [on github](https://github.com/openstax/openstax_cookbooks).  If you find yourself needing to modify these cookbooks, clone the cookbook repository and then set the ````OPENSTAX_COOKBOOKS_PATH```` environment variable to the absolute path of the checked out cookbooks before running vagrant commands.  This will cause Vagrant and Berkshelf to use your local copy of the OpenStax cookbooks instead of the ones on Github.
```

Comment:
If you select the text you'll see a couple of spaces at the end of this para that were removed.  Or add `?w=0` to this diff URL to see it go away.